testing/mocks/reader.go

```diff
@@ -20,13 +20,17 @@ import (
 )
 
 type Reader struct {
-	FirstFunc     func() (uint64, error)
-	LastFunc      func() (uint64, error)
-	HeaderFunc    func(height uint64) (*flow.Header, error)
-	CommitFunc    func(height uint64) (flow.StateCommitment, error)
-	EventsFunc    func(height uint64, types ...flow.EventType) ([]flow.Event, error)
-	RegistersFunc func(height uint64, paths []ledger.Path) ([]ledger.Value, error)
-	HeightFunc    func(blockID flow.Identifier) (uint64, error)
+	FirstFunc        func() (uint64, error)
+	LastFunc         func() (uint64, error)
+	HeaderFunc       func(height uint64) (*flow.Header, error)
+	CommitFunc       func(height uint64) (flow.StateCommitment, error)
+	EventsFunc       func(height uint64, types ...flow.EventType) ([]flow.Event, error)
+	RegistersFunc    func(height uint64, paths []ledger.Path) ([]ledger.Value, error)
+	HeightFunc       func(blockID flow.Identifier) (uint64, error)
+	TransactionFunc  func(transactionID flow.Identifier) (*flow.Transaction, error)
+	TransactionsFunc func(blockID flow.Identifier) ([]flow.Identifier, error)
+	CollectionFunc  func(collectionID flow.Identifier) (*flow.LightCollection, error)
+	CollectionsFunc func(blockID flow.Identifier) ([]flow.Identifier, error)
```

Comment:
Someone is not using `go fmt` :O

Reply:
Yep it's me, as I said yesterday this code was not final, I pushed a WIP here :p I run `go fmt` manually